server/main-api/src/maps/fetch_tile.rs

```diff
@@ -81,22 +81,28 @@ impl FetchTileTask {
         }
     }
 
-    async fn download_map_image(&self, file: &std::path::PathBuf) -> Option<web::Bytes> {
-        let tileserver_addr = std::env::var("MAPS_SVC_PORT_7770_TCP_ADDR")
-            .unwrap_or_else(|_| "localhost".to_string());
-        let tileserver_port = std::env::var("MAPS_SVC_SERVICE_PORT_TILESERVER")
-            .unwrap_or_else(|_| "7770".to_string());
-        let url = format!(
-            "http://{tileserver_addr}:{tileserver_port}/styles/osm_liberty/{z}/{x}/{y}@2x.png",
+    fn tileserver_url(&self) -> String {
```

Comment:
```suggestion
    fn get_tileserver_url(&self) -> String {
```
Only a minor thing but I think that makes it clear more quickly